src/styles/login.css

```diff
@@ -0,0 +1,43 @@
+#card {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    margin: auto;
+    width: 35%;
+    height: 50%;
```

Comment:
Making width/height percentages of screen size can lead to some strange looking elements on devices with different aspect ratios. For a form as simple as the login page, a good approach might be to just pick some fixed pixel sizes for height/width and center the component on the screen.

Flexbox is a good way to center elements both horizontally, and vertically.

Reply:
So I used a flexbox to center it and added height: 100vh to make the flexbox container take up the full screen. Not sure if this is a good way either though?